pkgs/tools/text/sgrep/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, sgrep, fetchurl, runCommand, lib, m4, makeWrapper }:
```

Comment:
Missing new line

Reply:
Ah, you mean a blank line between the arguments and the function body? Sure, let’s add it. Is this missing from the “coding conventions” section of the docs btw., or am I just overlooking it? If it’s missing I could try to add it later.